platforms-serverless-vercel/vercel-with-nextjs-caching/script2.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# to check that manual command removes caching error
```

Comment:
echo "running script.sh that manually runs 'prisma generate'"

Reply:
I think the log output of that would be swallowed by the package manager, or could show up in the vercel logs only.